### DISCUSSION TITLE
How to get consistent/readable font size for all clipboard items?

### DISCUSSION THREAD

**Discussion by andrewakl** [hluk/CopyQ#1830] [General]
Hi.
When i copy text from an HTML page with a small font size, the item shows up in the CopyQ main window with a small font size that is not easy to read.  Even when i try to increase the font size (Preferences -> Appearance -> Normal), the font size of the items that are text/plain increases, but the font size of the items that are text/html does not increase.

Is there another option, or future release, that will allow me to view the clipboard items with a consistent and readable font size?

![image](https://user-images.githubusercontent.com/59714327/142291494-220166fc-01eb-44cc-aa8a-aa51cd1aa45d.png)

Thanks.

**Comment by hluk** [comment]
The simplest solution is to [disable storing HTML](https://copyq.readthedocs.io/en/latest/faq.html#how-to-store-only-plain-text). If you still need to store the HTML format, just not display it, you will also have to add the following command ([here is how to add the command to CopyQ](http://copyq.readthedocs.io/en/latest/faq.html#how-to-load-shared-commands-and-share-them)):

```ini
[Command]
Automatic=true
Icon=\xf1c9
Input=text/html
Name=Store HTML
```

**Reply by andrewakl** [reply]
Thank you for your reply.
I was looking for a way to show the Html contents as plain text in the main window, preview (F7) as Html, and paste into the target window as Html.
It looks like this is not possible, so i will have to use the Windows Magnifier to zoom in whenever the contents are too small to read.

**Reply by hluk** [reply]
You can try enabling "Show simple items" in History config tab - this would show one-line plain text items in the main window without affecting the preview.

**Reply by andrewakl** [reply]
Thanks again.  Using "Show simple items" helped a little.
